SU-SOE/soe_basic
# D8CORE-2130: styles for the cta list

# READY FOR REVIEW

# Summary
- Updates to the Cta list paragraph

# Needed By (Date)
- Tuesday, 7th

# Urgency
- Med

# Steps to Test

1. Pull in this branch and the linked request below.
2. Verify when adding a CTA List or a story, the widths are limited like we do for the card.

# Affected Projects or Products
- SOE subtheming
- CTA List paragraph

# Associated Issues and/or People
-  D8CORE-2130
- https://github.com/SU-SOE/soe_basic/pull/8

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
